server.js

```diff
@@ -4,15 +4,16 @@ import matter from 'gray-matter'
 import { existsSync } from 'fsexists'
 import requireFromString from 'require-from-string'
 import { serialize } from 'next-mdx-remote/serialize'
+import { findComponent } from './utils/find-component'
 import components from './__swingset_components'
 
 export function createStaticPaths() {
   return function getStaticPaths() {
     return {
       paths: Object.values(components).map((componentConfig) => {
-        return { params: { swingset: [componentConfig.slug] } }
+        return { params: { swingset: ['components', componentConfig.slug] } }
       }),
```

Comment:
when we work in adding support for the standalone docs pages, we'll just have to make sure we prepend `'docs'` to the param array